opensplice_cmake_module/config/ros_ospl.xml

```diff
@@ -21,5 +21,14 @@
             <!-- the following one is necessary only for TwinOaks CoreDX DDS compatibility -->
             <!-- <ExplicitlyPublishQosSetToDefault>true</ExplicitlyPublishQosSetToDefault> -->
         </Compatibility>
+        <Discovery>
+             <!-- By default, OpenSplice only allows up to 9 participants (see
+                  http://download.prismtech.com/docs/Vortex/html/ospl/ConfGuide/guide.html#maxautoparticipantindex)
+                  This means that only 9 nodes can be run at a time, which even
+                  moderately complex robots easily achieve.  Increase this up to
+                  99 to allow many more nodes.
```

Comment:
I would suggest to remove the last sentence since it duplicates the following value.

Reply:
I did this and added wording about the potential downsides in 086154dde2505766cb75a9da7231ef44758dc2fa .  I still need to look into adding a test.

Reply:
Please move this additional information to the ticket description rather than the xml file.

Reply:
Done now.